fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/api/Persister.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.ocfl.api;
+
+import org.fcrepo.kernel.api.operations.ResourceOperation;
+import org.fcrepo.persistence.ocfl.impl.FedoraOCFLMapping;
+
+/**
+ * @param <T> The operation to be persisted.
+ *
+ * @author dbernstein
+ * @since 6.0.0
+ */
+public interface Persister<T extends ResourceOperation> {
+
+    /**
+     * The method returns true if the operation can be persisted by this persister.
+     * @param operation
+     * @return true or false
+     */
+    boolean handle(ResourceOperation operation);
+
+    /**
+     * The persistence handling for the given operation using the provided session and mapping.
+     * @param session The OCFL object session associated with the fedora resource
```

Comment:
"The session associated with the OCFL object" might be more accurate?

Reply:
Done.